examples/meanvar.jl

```diff
@@ -0,0 +1,44 @@
+# Welford's algorithm
+#
+# Welford, B. P. (1962). "Note on a method for calculating corrected sums of squares and products". Technometrics. 4 (3): 419–420. doi:10.2307/1266577. JSTOR 1266577
+
+
+using Transducers
+const T = Transducers
+
+struct MeanVar <: Transducer
+end
+
+
+function T.start(rf::T.R_{MeanVar}, result)
+    private_state = (0, 0.0, 0.0)
+    return T.wrap(rf, private_state, T.start(T.inner(rf), result))
+end
+
+
+function T.next(rf::T.R_{MeanVar}, result, input)
+    T.wrapping(rf, result) do st, iresult
+        (n, μ, M2) = st
+        n += 1
+        δ = input - μ
+        μ += δ/n
+        δ2 = input - μ
+        M2 += δ*δ2
+        iinput = (μ, M2 / (n-1))
+        iresult = T.next(T.inner(rf), iresult, iinput)
+        return (n,μ, M2), iresult
+    end
+end
+
+function T.complete(rf::T.R_{MeanVar}, result)
+    _private_state, inner_result = T.unwrap(rf, result)
+    return T.complete(T.inner(rf), inner_result)
+end
+
+collect(MeanVar(),randn(10))
+
+using BenchmarkTools
+@btime foldl(right, MeanVar(), 1:100)
```

Comment:
I want to run examples in the test so that I don't accidentally break them. Ideally, it's also nice to add it to the documentation. I rather not want to run benchmarks in those contexts.  If you want to add benchmarks, see e.g. `benchmark/bench_words.jl` for how to do it.  You also need to add `StatsBase` to `benchmark/Project.toml` and `benchmark/Manifest.toml`.

So, I suggest to remove `@btime` and just compare the returned value with `mean` and `var` (and not rely on `StatsBase`).